### PR TITLE
Add continuous integration badges

### PR DESCRIPTION
**Pull request by ssewilliam**
#### What does this PR do?
- Adds a Travis badge to the project

### CODE REVIEW

**Comment by AGMETEOR** [comment]
@ssewilliam is there a test coverage badge?

**Comment by ssewilliam** [comment]
@AGMETEOR coverage badge is not Included?

**Comment by pythonmomo** [comment]
@ssewilliam is it possible to have one?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # pythonmomo
 A simple to use python wrapper to integrate the MTN Uganda API into your projects.
 
+[![Build Status](https://travis-ci.org/pythonmomo/pythonmomo.svg?branch=master)](https://travis-ci.org/pythonmomo/pythonmomo)
+
+[![Coverage Status](https://coveralls.io/repos/github/pythonmomo/pythonmomo/badge.svg?branch=master)](https://coveralls.io/github/pythonmomo/pythonmomo?branch=master)
+
+### Just for test
+[![Coverage Status](https://coveralls.io/repos/github/ssewilliam/pythonmomo/badge.svg?branch=PYMOMO-0003)](https://coveralls.io/github/ssewilliam/pythonmomo?branch=PYMOMO-0003)
```
Comment (pythonmomo) [inline README.md]: @ssewilliam what's this one for?

Comment (ssewilliam) [reply README.md]: Just for testing would be removed

**Pull request merged**